src/patient-registration/patient-registration.component.tsx

```diff
@@ -71,7 +71,7 @@ export interface FormValues {
   relationships: Array<{ relatedPerson: string; relationship: string }>;
 }
 
-export const initialFormValues: FormValues = {
+export var initialFormValues: FormValues = {
```

Comment:
This is odd. What's going on here? How is this export used?

Reply:
export was already present there. Why I changed const to var is because, whenever you got to the edit url, existingPatient variable on line 173, has the details of the patient and initialFormValues properties are set according to the existingPatient details as you can see after line no. 173. 

When you go from /edit page to /patient-registration page, the existingPatient variable has the patientDetails and after some delay. the context sets the existingPatient details to null.

When existingPatient becomes null, then the values of intialFormValues is also needed to set to intial values, otherwise the form is filled up with the last patient who's details were going to be editted. Hence I added the else statement, to re-assign the initial values to initialFormValues.

Reply:
Right, sorry, what I meant to ask is, why is it a `var` now? It probably shouldn't be a `var`—that suggests some kind of externally-accessible internal state without constraints, which we should avoid.

Reply:
(always prefer `let` to `var`, anyway)

Reply:
Actually if you see the whole code, we are using `initialFormValues` in the form. When we are getting the current patient, we are updating the properties of initialFormValues as you can see the code below.

The issue that comes up is, after opening the edit page of a patient, we go to the /patient-registration, the values are already filled up with the initialFormValues that were changed earlier.

Hence, we need a check for the 
if(currentPatient){
initalFormValues'properties are changed
}Else{
Reset the initialFormValues
}

I added the else clause. Since we cannot assign a constant, I changed it to var for the purpose.

Reply:
I'll push it with `let`.

Reply:
Are you clear what I wanted to convey @brandones ?

Reply:
@brandones  I have made some changes to reset the initialFormValues in a new way. Can you have a look into it. I hope this approach is way more clear.
Please review the new changes.
Thank You!